### DISCUSSION TITLE
Prism.Maui proper project setup

### DISCUSSION THREAD

**Discussion by brzezinol** [PrismLibrary/Prism#2865] [.NET MAUI]
I run into issue that I can't properly run Prism.Maui.
When I create project from template, project not work properly.
When I try to cretate project targeting .net7.0, I cannot fingld nuget which is compatible with .net7.
How it is with Prism.Maui?

**Comment by giuseppenovielli** [comment]
Hi @brzezinol,
check my project sample MAUI7 -> https://github.com/giuseppenovielli/MAUI_XF-Prism-Full-Navigation-Example/tree/master

**Comment by dansiegel** [comment]
currently you must be a GitHub sponsor to access the version of Prism.Maui that supports .NET 7 we will be releasing an update publicly however I would like to try to get #2815 fixed before we publish that. If someone from the community would like to submit a PR to fix it, that would help me to publish a new preview to nuget.
